app/renderer/components/Inspector/shared.js

```diff
@@ -52,3 +52,115 @@ export const SCREENSHOT_INTERACTION_MODE = {
   SWIPE: 'swipe',
   TAP: 'tap',
 };
+
+export const actionArgTypes = {
+  STRING: 'string',
+  NUMBER: 'number',
+};
+
+const { STRING, NUMBER } = actionArgTypes;
+
+export const actionDefinitions = {
+  'Device': {
```

Comment:
should these action definition captions be translated?

Reply:
Not here, but inside the React component, yes.